### DISCUSSION TITLE
compile for wemos_d1_mini32 8266, how?

### DISCUSSION THREAD

**Discussion by bwired-nl** [tbnobody/OpenDTU#204] [General]
what do i need to set in the platform.ini for compiling on a wemos d1 mini
I see the settings for D1 mini board , made some changes in generic section etc. but it keeps compiling ESP32?
thanks

**Comment by ghost** [comment]
The OpenDTU project is **only** intended for ESP32 boards. In a comparable project (https://github.com/lumapu/ahoy) ESP8266 boards can also be used.

If you want to use an ESP32 D1 **mini** board, you have to select the ESP32 D1 mini in PlatformIO under "Switch PlatformIO Project Environment".


![PlatformIO_env_wechseln](https://user-images.githubusercontent.com/107404965/194759549-63229e3d-b367-4714-878f-73571e8ec9d2.png)
